MapboxVision/src/main/java/com/mapbox/vision/VisionManager.kt

```diff
@@ -292,6 +302,7 @@ object VisionManager : BaseVisionManager {
 
         locationEngine.detach()
         sensorsManager.detach()
```

Comment:
Mind adding an issue to reuse `Attachable` for other components here, too? Or doing in in this PR?

Reply:
I'd like to avoid do it in this PR, so I'm gonna create issue for it

Reply:
K!